project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/directives/types/types.html

```diff
@@ -1,3 +1,3 @@
-<div class="form-group btn-group btn-group-{{field}}">
-  <button type="button" ng-value="value.id" ng-class="{active: value.id==type}" ng-model="type" ng-required="!discussionType" ng-click="updateType(value.id, field)" class="btn btn-default {{field}} col-xs-12 col-lg-{{cols}}" ng-repeat="value in allowedValues"><i class="fa"> {{ value.label }}</button>
+<div class="form-group btn-group-{{field}}">
+  <button type="button" ng-value="value.id" ng-class="{active: value.id==type}" ng-model="type" ng-required="!discussionType" ng-click="updateType(value.id, field)" class="btn btn-default {{field}} col-xs-12 col-lg-{{cols}}" ng-repeat="value in allowedValues"><i class="fa"></i> {{ value.label }}</button>
```

Comment:
This is how it was implemented before, since the content is generated dynamically. 
I can dig in and try to find a way to put the correct icon class name directly, if you want.